src/main/java/com/paneedah/mwc/entities/Entities.java

```diff
@@ -39,49 +42,55 @@ public static void init(CommonProxy commonProxy) {
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityEnderman.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityCreeper.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntitySlime.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityDragon.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityEndermite.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityBlaze.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityGhast.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityIllusionIllager.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityMagmaCube.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityPigZombie.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntitySpellcasterIllager.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityStray.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityVindicator.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityWitch.class, true))
+                .withAiTargetTask(3, e -> new EntityAINearestAttackableTarget<>( e, EntityZombieVillager.class, true))
                 .register(MWC.modContext);
 
         new EntityConfiguration.Builder()
                 .withName("soldier")
                 .withBaseClass(EntityCustomMob.class)
-                .withMaxHealth(80)
+                .withMaxHealth(30)
                 .withEntityIdSupplier(() -> 10011)
                 .withEquipmentOption(Guns.M4A1, EnumDifficulty.EASY, 0.1f, Magazines.M4A1Mag)
                 .withEquipmentOption(Guns.M110, EnumDifficulty.EASY, 0.1f, Magazines.M110Mag)
+                .withEquipmentOption(Guns.M16A1, EnumDifficulty.NORMAL, 0.2f, Magazines.M4A1Mag)
+                .withEquipmentOption(Guns.FNFAL, EnumDifficulty.NORMAL, 0.1f, Magazines.FNFALMag)
+                .withEquipmentOption(Guns.M4A1, EnumDifficulty.HARD, 0.2f, Magazines.Stanag100)
+                .withEquipmentOption(Guns.M249, EnumDifficulty.HARD, 0.1f, Magazines.M249Mag)
+                .withSecondaryEquipmentOption(Guns.Glock18C,  EnumDifficulty.NORMAL, 0.1f, Magazines.M9A1Mag)
+                .withSecondaryEquipmentOption(Guns.DesertEagle,  EnumDifficulty.HARD, 0.1f, Magazines.DesertEagleMag)
                 .withPrimaryEquipmentDropChance(0.4f)
+                .withMaxAmmo(1500)
```

Comment:
AI already has max ammunition, by default is it 36000

Reply:
> AI already has max ammunition, by default is it 36000

Which is vitrually infinite since they shoot about once every second.
Anyway irrelevant, I did not make this, but they should have infinite ammunition, they shouldn't check against a max ammount, they should just shoot if they desire so.